jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC

```diff
@@ -33,25 +31,21 @@ export EXT=""
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${NET}.${PDY})/${cyc}/${COMPONENT}/gempak}
-
-export SENDDBN=${SENDDBN:-NO}
-export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
-
-mkdir -m 775 -p ${COMOUT}
-
+export COMIN="${COMIN:-$(compath.py "${envir}"/"${NET}"/"${gfs_ver}")/${RUN}.${PDY}/${cyc}/${COMPONENT}}"
+export COMOUT="${COMOUT:-$(compath.py -o "${NET}"/"${gfs_ver}"/"${NET}"."${PDY}")/${cyc}/${COMPONENT}/gempak}"
 
-# TODO - Assess what is going on with overwriting $DATA here (#1224)
+export SENDDBN="${SENDDBN:-NO}"
+export DBNROOT="${DBNROOT:-${UTILROOT}/fakedbn}"
 
-export DATA_HOLD=${DATA}
+mkdir -m 775 -p "${COMOUT}"
 
 #################################################################
 # Execute the script for the regular grib
 #################################################################
-export DATA=${DATA_HOLD}/SPECIAL
-mkdir -p ${DATA}
-cd ${DATA}
+export DATA_SPECIAL="${DATA}/SPECIAL"
+mkdir -p "${DATA_SPECIAL}"
+# shellcheck disable=SC2164
```

Comment:
Don't need to disable SC2164 here...can address that shellcheck code by doing what is suggested here: https://www.shellcheck.net/wiki/SC2164

Reply:
@KateFriedman-NOAA I didn't muck with resolving all of the linter complaints here. I don't have a way to check whether they will break the code. I am happy to look into this further, but I need a known configuration against which I can test such that any modifications I make do not break this. Similar to other instances, if this PR clears I am happy to open another issue/PR to address them once I can test things further.

Reply:
Totally understand the concern but I don't think adding `|| exit` will break anything. If it does happen to break things when eventually tested then the developer who comes in to get these jobs working can resolve it. Would prefer to make small shellcheck fixes like this now than put in disables that could get missed by a follow-up developer.

Please make the changes to satisfy SC2164 (here and in other spots I noted). Thanks!

Reply:
@KateFriedman-NOAA I apologize, I missed some of these probably because I am trying to wrangle too PRs (and need coffee).

They should be resolved. Thank you for catching them.